Node playground documentation enhancements (#2832)
foxglove/studio: packages/studio-base/src/panels/NodePlayground/index.help.md

> Write nodes that manipulate, reduce, and filter existing messages and output them to other Studio panels.

Reviewer: Would still like some reference to `topic`s, since that is how users will find them in Studio panels.

> Write nodes that manipulate, reduce, and filter existing messages on topics and output them to new topics for other Studio panels.

Reviewer 2: ```suggestion
Write code to manipulate, aggregate, and filter existing messages on topics and output them to new topics for other Studio panels to visualize.
```

_Node Playground_ scripts are written in [TypeScript](https://www.typescriptlang.org/).

> When you create a new node, you’ll be presented with some boilerplate to get started. Every node must declare 3 exports:

Reviewer 2: ```suggestion
When you create a new node, you’ll be presented with some boilerplate to get started. Every node must declare 3 [exports](https://www.typescriptlang.org/docs/handbook/modules.html#export):
```

- `inputs`: an array of topic names
- `output` topic for the output messages of your node

> - `node` function that takes messages from input topics and returns new messages

Reviewer: Use formatting consistent with rest of docs. 

- `inputs` – An array of topic names
- `output` – Topic for your node's output messages
- `node` - Function that takes your `inputs` and publishes new messages on your `output`

> - `inputs` – An array of topic names
> - `output` – Topic for your node's output messages
> - `node` – Function that takes your `inputs` and publishes new messages on your `output`

Reviewer 2: ```suggestion
- `inputs: string[]` – An array of topic names
- `output: string` – Topic for your node's output messages
- `node: (event: MessageEvent, variables: {}) => unknown` – Function that takes your `inputs` and publishes new messages on your `output`
```

Check out the _templates_ within the editor for sample nodes.

> To debug your code, call `log(someValue, anotherValue)` to print non-function values to the Logs section at the bottom of the editor panel.

Reviewer 2: "non-function" is kind of a weird thing to call out
```suggestion
To debug your code, call `log(someValue, anotherValue)` to print values to the Logs section at the bottom of the editor panel.
```
